### DISCUSSION TITLE
Lab Assignment #4 👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#18] [Assignments]
## Due 5/16 before class 

Please use this thread to post your [lab assignment's #4](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week5/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/23S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### For example

https://albertkun.github.io/23s-asiaam-191a-sample/week5/index.html

**Comment by czhangy** [comment]
https://czhangy.github.io/ASIA-AM-191A/week4/maps.html

**Reply by ghost** [reply]
Nicely done @czhangy ! But I should mention, that you need to make your survey viewable for everyone! 
![image](https://user-images.githubusercontent.com/57691411/236368834-92c20264-5b8b-4fe3-a4e7-2b576205069e.png)

**Comment by zacharypakin** [comment]
https://zacharypakin.github.io/aa191a-23s/week05/maps.html

**Reply by zacharypakin** [reply]
updated with new survey

**Comment by shintuzd** [comment]
https://shintuzd.github.io/23s-asiaam-191a-assignments/week5/maps.html

**Comment by paige-lee** [comment]
https://paige-lee.github.io/aa191a-labs/week5/index.html

**Comment by qatle** [comment]
https://trinhqale.github.io/AM191A-labs/week5/index.html

**Comment by SageLuong** [comment]
https://sageluong.github.io/23s-asianam-191a-labs/week5/animals.html

**Comment by cassidytu** [comment]
https://cassidytu.github.io/asam191a-labs/week5/map.html

**Comment by joanneluong** [comment]
https://joanneluong.github.io/aas191a-labs/week5/index.html

**Comment by chrysanthemma** [comment]
https://chrysanthemma.github.io/asainam191a-labs/Lab4/index.html

**Comment by vickydelk** [comment]
https://vickydelk.github.io/aa191-assignments/week5/index.html

**Comment by angelicaalacantar** [comment]
https://angelicaalacantar.github.io/aas191a-23s/week6/index.html

**Comment by danigold1020** [comment]
https://danigold1020.github.io/23s-asianam-191a-labs/week05/index.html

**Comment by catherinehhu** [comment]
https://catherinehhu.github.io/191a-labs/week5/index.html

**Comment by ariscorpus** [comment]
https://ariscorpus.github.io/23s-asianam-191a-labs/week5(b)/index.html

**Comment by jamesfalkenroth** [comment]
https://jamesfalkenroth.github.io/23s-asianamerican-191a-labs/week5/index.html

**Comment by cassiuscarandang** [comment]
https://cassiuscarandang.github.io/23s-asiaam-191a-lab1/week5lab/

**Comment by DylanJWan** [comment]
https://dylanjwan.github.io/asiaam-191a-assignments/week5/index.html

**Comment by maizaha** [comment]
https://maizaha.github.io/23s-asiaam-191a-labs/week5/index.html

**Comment by kmtsujimoto** [comment]
https://kmtsujimoto.github.io/23s-asiaam-191a-labs/week05/index.html

**Comment by enyi-emuka** [comment]
https://enyi-emuka.github.io/asiaam191a_labs/week5/index.html

**Comment by trufflelover** [comment]
https://trufflelover.github.io/23s-asiaam-191a-labs/lab4/

**Comment by JonathanGLopez** [comment]
http://127.0.0.1:5500/week5/index.html

**Reply by JonathanGLopez** [reply]
not a GitHub link
Git Hub link 
https://jonathanglopez.github.io/Asiaam191aLB1/week5/index.html

**Comment by mandymwlai** [comment]
https://mandymwlai.github.io/AA-191-lab/week5/

**Comment by stellakangj** [comment]
https://stellakangj.github.io/asiaam191a/week%205/maps.html

**Comment by alanamalika** [comment]
https://alanamalika.github.io/23s-asianam-191a-labs/week05/index.html

**Comment by amandatlxu** [comment]
https://amandatlxu.github.io/23SP-AsianAm191A-Labs/week4/index.html

**Comment by rebeccaksuen** [comment]
https://rebeccaksuen/github.io/23s-asiaam-191a-labs/week5/index.html

**Reply by rebeccaksuen** [reply]
https://rebeccaksuen.github.io/23s-asiaam-191a-labs/week5/index.html
